### DISCUSSION TITLE
Download button does not appear on mobile view

### DISCUSSION THREAD

**Discussion by guzelonur** [brackets-cont/brackets#164] [General]
Hi,

Just noticed that when you go to [brackets.io](http://brackets.io)  through your any Chromium mobile browser, the download button does not appear as I had much difficulty finding the download button until switching to desktop version.

It must be a kind of incorrect responsive site design.

Thanks!

**Comment by kirtisingh3008** [comment]
> Hi,
> 
> Just noticed that when you go to [brackets.io](http://brackets.io) through your any Chromium mobile browser, the download button does not appear as I had much difficulty finding the download button until switching to desktop version.
> 
> It must be a kind of incorrect responsive site design.
> 
> Thanks!

Hi, Brackets is not available on mobile devices, so we don't display the download button there. It is by design only.
Thanks!
